backend/spellchecker/server/parser.py

```diff
@@ -0,0 +1,70 @@
+# Author: Sergey Sokolov
+from ctypes import cdll, c_char_p, pointer, Structure
+
+
+class Parser:
+    """
+    Class wraps CFFI for LaTeX parser from Hunspell source code.
+    """
+    def __init__(self):
+        # note: these fields' names doesn't start with "_" since deleting of this 
+        # object isn't so straightforward -- for some reason interpreter destroys 
+        # these "private" fields before __del__ is invoked.
+        self.parser_lib_ = cdll.LoadLibrary("../libparser/build/libparser.so")
+        self.parser_ = self.parser_lib_.latex_parser()
+
+    def __del__(self):
+        self.parser_lib_.delete_parser(self.parser_)
+
+    def put_line(self, line: str):
```

Comment:
Actually, no, it's just named after parser's method name, but it's okay with linebreaks, I checked that